addon/components/sl-calendar-month.js

```diff
@@ -81,11 +73,14 @@ export default Ember.Component.extend({
     /**
      * The short string name of the represented month
      *
-     * @function shortName
-     * @returns  {Ember.String}
+     * @function
+     * @observes month
```

Comment:
It's planned for the future in the same way that the of use JSDoc is.  Once we are actually using it to generate the documentation the https://github.com/notmessenger/jsdoc-plugins#emberobservestag plugin will be leveraged as well.